src/apps/dfx/ethereum/dfx.amm.token-fetcher.ts

```diff
@@ -0,0 +1,103 @@
+// Test address: http://localhost:5001/apps/dfx/tokens?groupIds[]=dfx-curve&network=ethereum
+//
+import { Inject } from '@nestjs/common';
```

Comment:
The name of this file should include the `groupId`, so you can instead make it `dfx.curve.token-fetcher.ts`.

Reply:
Fix https://github.com/Zapper-fi/studio/pull/347/commits/e25acbfb6a54a1f8e8cd59aa77be53317e37f202